lib/elixir/src/elixir_try.erl

```diff
@@ -57,9 +57,11 @@ each_clause({Key, Meta, _, _}, S) ->
 build_rescue(Meta, Parts, Body, S) ->
   Matches = [Match || {Match, _} <- Parts],
 
-  {{clause, Line, TMatches, _, TBody}, TS} =
+  {{clause, Line, TMatches, _, TBody}, TSTmp} =
     elixir_clauses:clause(Meta, fun elixir_translator:translate_args/2,
-                          Matches, Body, [], S),
+                          Matches, Body, [], S#elixir_scope{extra=nil}),
+
+  TS = TSTmp#elixir_scope{extra=S#elixir_scope.extra},
```

Comment:
Do you know why we need to change it here if we already changed it here? https://github.com/elixir-lang/elixir/pull/4504/files#diff-dd6727eaeac58aaeae8719869f854c8bR105